dockerstack/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+services:
+  fileserver:
+    build: ./fileserver
+    ports:
+      - "8081:80"
+    env_file:
+      - vars.env
```

Comment:
We aren't using these vars in this container. I think I just included it as an example.

```suggestion
```

Reply:
Done.